docusaurus/docs/engineering/boba-backend/APIs/intro.md

```diff
@@ -4,23 +4,23 @@ sidebar_position: 1
 
 # Introduction to APIs
 
-APIs allow applications (clients) to interact with another application (server) without unfettered access to the underlying data, or to the whole set of operations that can be executed on it. In short, an API is a contract defined by the server that establishes a common language that clients can use to communicate with it. Most operations allowed by APIs boil down to fetching, storing or updating data.
+APIs allow different applications to talk to each other without giving them complete access to all the data or operations of the other application. In simple terms, an API is a contract that defines a common language so one application can communicate with another. Most API operations involve fetching, storing, or updating data.
```

Comment:
We probably should still add a sentence about client/server here

Reply:
I agree. Maybe

"Most API operations involve fetching, storing, or updating data when a client sends a request to a server."